app/views/steps/respondent/has_other_parties/edit.html.erb

```diff
@@ -1,15 +1,15 @@
 <% title t('.page_title') %>
+<% step_header %>
 
-<div class="grid-row">
-  <div class="column-two-thirds">
-    <%= step_header %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= govuk_error_summary %>
 
-    <h1 class="heading-xlarge gv-u-heading-xxlarge"><%=t '.heading' %></h1>
+    <h1 class="govuk-heading-xl"><%=t '.heading' %></h1>
+    <p class="govuk-body"><%=t '.info_html' %></p>
 
-    <p><%=t '.info_html' %></p>
-
-    <%= step_form @form_object do |f| %>
-      <%= f.radio_button_fieldset :has_other_parties, inline: true, choices: GenericYesNo.values, legend_options: { class: 'visually-hidden', text: t('.heading') } %>
+    <%= step_form @form_object, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+      <%= f.govuk_collection_radio_buttons :has_other_parties, GenericYesNo.values, :value, nil, legend: { tag: 'span', size: 'm' } %>
```

Comment:
Let's move the `info_html` as injected content, so the fieldset legend can use the header, size XL.

Like here pretty much:
https://github.com/ministryofjustice/c100-application/blob/develop/app/views/steps/international/resident/edit.html.erb

Otherwise this fails accessibility (that's why the visually hidden in the old version, so the radio fieldset legend had the heading of the page, but visually hidden).